### DISCUSSION TITLE
Can't find variable: queueMicrotask

### DISCUSSION THREAD

**Discussion by thanhtoan1196** [livewire/livewire#3831] [7. Bugs (archived)] [answered]
I get a lot of these errors from Sentry, how do I fix it?

![image](https://user-images.githubusercontent.com/16433547/134151554-3ea4c520-df58-4e06-965b-e5da70f3c521.png)

![image](https://user-images.githubusercontent.com/16433547/134151794-7f783906-824b-4cd5-92e9-3454293f6052.png)

**Comment by mabdullahsari** [comment, marked as answer]
You "can not". Those browsers are probably ancient. If Livewire does not provide a polyfill (which is likely), you have to polyfill it yourself.

```js
if (typeof window.queueMicrotask !== 'function') {
    window.queueMicrotask = cb => Promise.resolve().then(cb);
}
```

**Reply by thanhtoan1196** [reply]
ok, thank you

**Comment by guoapeng** [comment]
thanks @mabdullahsari, that helped me and inspired me on finding a better solution for such kinds of issues for my angular project.
actually core-js provided a bunch of features to polyfill the gap between newer js syntax, functions, classes and older browser runtime environment. and I just added the following line in the polyfill.ts and the problem was gone.

polyfill.ts
```ts
import "core-js/modules/web.queue-microtask"
```

core-js also provider other import syntax, please refer to its official github repo https://github.com/zloirock/core-js
